feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/SessionPageFragment.kt

```diff
@@ -124,6 +125,11 @@ class SessionPageFragment : DaggerFragment() {
                 bottomSheetBehavior.state = BottomSheetBehavior.STATE_EXPANDED
             }
         }
+        sessionPagesStore.tabState.changed(viewLifecycleOwner) {
+            if (it == ViewPager.SCROLL_STATE_IDLE) {
+                bottomSheetBehavior.state = BottomSheetBehavior.STATE_EXPANDED
+            }
```

Comment:
at first I wanted to set `BottomSheetBehavior.isDraggable = false` for `ViewPager.SCROLL_STATE_DRAGGING` here.
but the time is too short and BottomSheet down action is already triggered.. so it's not worked 😢